docs/Users/Tutorials/Auto_Completion.rst

```diff
@@ -0,0 +1,38 @@
+Auto completion
+===============
+
+If you're using bash or zsh you can setup them to have coala auto command
+completion.
+
+Bash
+----
+
+You have to install ``argcomplete``
+
+::
+
+    $ pip install argcomplete
+
+After this you have to either activate it
+`globally <https://github.com/kislyuk/argcomplete#activating-global-completion>`__
+or add
+
+::
+
+    eval "$(register-python-argcomplete `which coala`)"
+
+to your ``.bashrc``. Just make sure that coala is added to the path at that
+point. If you want, or you are unsure if coala is in the PATH, you can expand
+``\`which coala\``` to the path of your coala executable (you can find where it
+is by running ``which coala``.
+
+Zsh
```

Comment:
ok

Sincerely,

Lasse Schuirmann

contact@viperdev.io
http://viperdev.io/

2016-04-30 14:51 GMT+02:00 Lasse Schuirmann <lasse.schuirmann@gmail.com>:

>
>
> Sincerely,
>
> Lasse Schuirmann
>
> contact@viperdev.io
> http://viperdev.io/
>
> 2016-04-30 14:39 GMT+02:00 Attila <notifications@github.com>:
>
>> In docs/Users/Tutorials/Auto_Completion.rst
>> <https://github.com/coala-analyzer/coala/pull/2097#discussion_r61669178>:
>>
>> > +    $ pip install argcomplete
>> > +
>> > +After this you have to either activate it
>> > +`globally <https://github.com/kislyuk/argcomplete#activating-global-completion>`__
>> > +or add
>> > +
>> > +::
>> > +
>> > +    eval "$(register-python-argcomplete `which coala`)"
>> > +
>> > +to your ``.bashrc``. Just make sure that coala is added to the path at that
>> > +point. If you want, or you are unsure if coala is in the PATH, you can expand
>> > +``\`which coala\``` to the path of your coala executable (you can find where it
>> > +is by running ``which coala``.
>> > +
>> > +Zsh
>>
>> I think it's OK. Look here http://www.zsh.org/.
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/coala-analyzer/coala/pull/2097/files/1c98d049c64659e7cd3b3afbf2bf8573a17298bf..3e542e1ef579f5e5fd17a68d62eb54e353e1bbfc#r61669178>
>>
>
>